src/main/resources/migrations/00005_drop_not_null_transaction_table.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE transaction ALTER COLUMN gateway_account_id DROP NOT NULL;
```

Comment:
Can we add the usual liquibase markings, eg `--liquibase formatted sql` and assign a liquibase changeset id eg `--changeset uk.gov.pay:drop_not_null_transaction_table` ? I've seen unmarked files wreck havoc when changeset id is missing.

Reply:
Thanks, fixed